### PR TITLE
[RHOSINFRA-154] Fix Babel in requirements.

### PR DESCRIPTION
**Pull request by obaranov**
@yfried-redhat, can you please review?

### CODE REVIEW

**Comment by tkammer** [comment]
The problem is specific for Fedora23, I think we should add a Doc comment that Fedora23 is currently problematic.

**Comment by obaranov** [comment]
I think the problem is not only with Fedora 23. The problem is in the python-novaclient**<3** IR requirement. 

If installs 2.3.3 Babel. And this version is not supported by the recent python-heatclient package (heaclient has not restrictions)

**Comment by yfried-redhat** [comment]
retest openstack

**Comment by yfried-redhat** [comment]
retest virsh

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,9 @@
...
-ansible>=2.0.0
+shade>=1.7.0
+python-novaclient>=2.21.0,!=2.27.0,!=2.32.0
+ansible>=2.0.0,!=2.0.2
 clg>=2.0.0
 colorlog>=2.6.1
 ipaddress==1.0.16
-python-novaclient<3
 PyYAML>=3.11
 shade>=1.7.0
```
Comment (yfried-redhat) [inline requirements.txt]: shade is here twice

**Comment by yfried-redhat** [comment]
retest openstack

**Comment by yfried-redhat** [comment]
retest openstack

**Comment by yfried-redhat** [comment]
retest openstack

File: requirements.txt [hunk 2]
```diff
@@ -1,10 +1,12 @@
-ansible>=2.0.0
+pip>=8.0.0
```
Comment (yfried-redhat) [inline requirements.txt]: I don't think this means anything. AFAIK, the new pip will be active only after all packages were installed.
Let's try it without this line

**Pull request merged**